interchaintest/go.mod

```diff
@@ -12,36 +12,38 @@ replace (
 	// github.com/CosmosContracts/juno => ../../
 	github.com/CosmosContracts/juno/v16 => github.com/CosmosContracts/juno/v16 v16.0.0-20230524152917-465dae509bfe
 	github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.2 //indirect
+	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.47.3
```

Comment:
Does the SDK have to be pinned to 47.3 for POB integration to work? - We are good with bumping it since this is just a test